notebooks/community/ml_ops/stage6/get_started_with_custom_image_model_batch.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<p>Is it necessary here? If not, please specify so users can skip if they want. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/915/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>the prediction container is used by batch prediction</p>